bin/alex_model.inl

```diff
@@ -1,289 +0,0 @@
-// Alex's Model
```

Comment:
I do expect only CMake part update. If you want to update the package version from upstream I need to create separate branch for that. What upstream release tag are you interested in?

Reply:
i've used https://github.com/liuliu/ccv/tree/stable

Reply:
> What upstream release tag are you interested in?

Okay, it seems they don't have release tags unfortunately. I have created branch `hunter-0.7-07fc691` based on 07fc691 commit, please rebase this pull request accordingly.